ospfd/ospf_vty.c

```diff
@@ -9871,7 +9868,7 @@ static int config_write_network_area(struct vty *vty, struct ospf *ospf)
 			/* Create Area ID string by specified Area ID format. */
 			if (n->area_id_fmt == OSPF_AREA_ID_FMT_DOTTEDQUAD)
 				strncpy((char *)buf, inet_ntoa(n->area_id),
```

Comment:
Same `strncpy` comment applies here.

Reply:
You may want to do the same thing as suggested here: https://github.com/FRRouting/frr/pull/2435#discussion_r195418264

> Looks good to me, but I'd prefer one of the following:
>
> ```c
> strlcpy(area_id, inet_ntoa(area->area_id), sizeof(area_id));
> inet_ntop(AF_INET, &area->area_id, area_id, sizeof(area_id));
> ```
>
> strncpy() is tricky and we should try to avoid it whenever we can.